ext/odbc/tests/odbc_fetch_object_001.phpt

```diff
@@ -0,0 +1,46 @@
+--TEST--
+odbc_fetch_object(): Getting data from query
+--EXTENSIONS--
+odbc
+--SKIPIF--
+<?php include 'skipif.inc'; ?>
+--FILE--
+<?php
+
+include 'config.inc';
+
+$conn = odbc_connect($dsn, $user, $pass);
+
+odbc_exec($conn, 'CREATE TABLE fetch_object (foo INT)');
+odbc_exec($conn, 'INSERT INTO fetch_object VALUES (1)');
+odbc_exec($conn, 'INSERT INTO fetch_object VALUES (2)');
+odbc_exec($conn, 'INSERT INTO fetch_object VALUES (3)');
+
+$res = odbc_exec($conn, 'SELECT * FROM fetch_object');
+
+var_dump(odbc_fetch_object($res, 1));
+var_dump(odbc_fetch_object($res));
+var_dump(odbc_fetch_object($res, 0));
```

Comment:
Does it just ignore a value of ``0`` and fetch the next entry?

Reply:
Yes, this is the relevant piece of code:

```
if (rownum > 0) {
    rc = SQLExtendedFetch(result->stmt,SQL_FETCH_ABSOLUTE,rownum,&crow,RowStatus);
} else {
    rc = SQLExtendedFetch(result->stmt,SQL_FETCH_NEXT,1,&crow,RowStatus);
}
```

The default values of odbc_fetch_*() functions are highly inconsistent, but I'll fix them with my resource to object converter PR (which is basically done, but needs much more code coverage, thus this PR).